flink-examples/flink-examples-streaming/src/main/java/org/apache/flink/streaming/examples/elasticsearch/Elasticsearch1Example.java

```diff
@@ -37,20 +39,36 @@
 import java.util.Map;
 
 /**
- * This example shows how to use the Elasticsearch Sink. Before running it you must ensure that
- * you have a cluster named "elasticsearch" running or change the cluster name in the config map.
+ * End to end test for elasticsearch1.
+ *
+ * <p>This example shows how to use the Elasticsearch Sink from an user endpoint. Before running it you
+ * must ensure that you have a cluster named "elasticsearch" running or change the cluster name in the config map.
+ *
+ * <p>Example usage:
+ * 	--index my-index-person --type my-type-spiderman
  */
-@SuppressWarnings("serial")
-public class ElasticsearchSinkExample {
+public class Elasticsearch1Example {
```

Comment:
I think we should include the word "Sink" in the class name.

Reply:
will update.